app/controllers/curation_concerns/single_use_links_viewer_controller.rb

```diff
@@ -0,0 +1,21 @@
+module CurationConcerns
+  class SingleUseLinksViewerController < ApplicationController
+    include CurationConcerns::SingleUseLinksViewerControllerBehavior
+
+    class Ability
+      include CanCan::Ability
+
+      attr_reader :single_use_link
+
+      def initialize(user, single_use_link)
+        @user = user || ::User.new
+
+        @single_use_link = single_use_link
+
+        can :read, ActiveFedora::Base do |obj|
+          single_use_link.valid? && single_use_link.itemId == obj.id && single_use_link.destroy!
+        end if single_use_link
```

Comment:
My $0.02is not to sweat the rubocopisms 'til #152 is merged (not ready yet). I'll rebase on master and take care of all offenses in one fell swoop, so no worries here.